.github/workflows/pm_stale_ticket_labeler.yml

```diff
@@ -0,0 +1,26 @@
+name: Label inactive issues and pull requests
+on:
+  schedule:
+    - cron: "12 2 * * *"
+
+jobs:
+  label-stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          start-date: 2025-01-01
+          exempt-issue-labels: 'no-automation,"Type: Epic"'
+          exempt-pr-labels: no-automation
+          exempt-assignees: jimchamp,mekarpeles,cdrini,scottbarnes
+          days-before-close: -1  # `-1` signifies that neither issues nor pull requests are to be closed
+          days-before-stale: 14
+          include-only-assigned: true
+          stale-issue-label: "Needs: Review Assignee"
+          stale-pr-label: "Needs: Review Assignee"
+          stale-issue-message: 'This issue has been marked "Needs: Review Assignee" since there have been no updates within the past two weeks'
+          stale-pr-message: 'This PR has been marked "Needs: Review Assignee" since there have been no updates within the past two weeks'
```

Comment:
```suggestion
          days-before-issue-stale: 14
          days-before-pr-stale: -1
          include-only-assigned: true
          stale-issue-label: "Needs: Review Assignee"
          stale-issue-message: 'This issue has been marked "Needs: Review Assignee" since there have been no updates within the past two weeks'
```